doc/classes/Basis.xml

```diff
@@ -148,13 +181,33 @@
 			<param index="1" name="angle" type="float" />
 			<description>
 				Introduce an additional rotation around the given axis by [param angle] (in radians). The axis must be a normalized vector.
+				[codeblock]
+				var my_basis = Basis.IDENTITY
+				var angle = PI / 2
+
+				my_basis = my_basis.rotated(Vector3.UP, angle)      # Rotate around the up axis (yaw)
+				my_basis = my_basis.rotated(Vector3.RIGHT, angle)   # Rotate around the right axis (pitch)
+				my_basis = my_basis.rotated(Vector3.FORWARD, angle) # Rotate around the forward axis (roll)
+				[/codeblock]
 			</description>
 		</method>
 		<method name="scaled" qualifiers="const">
 			<return type="Basis" />
 			<param index="0" name="scale" type="Vector3" />
 			<description>
 				Introduce an additional scaling specified by the given 3D scaling factor.
+				[codeblock]
+				var my_basis = Basis(
+				    Vector3(1, 1, 1),
+				    Vector3(2, 2, 2),
+				    Vector3(3, 3, 3)
+				)
+				my_basis = basis.scaled(Vector3(0, 2, -2))
+
+				print(my_basis.x) # Prints (0, 2, -2).
+				print(my_basis.y) # Prints (0, 4, -4).
+				print(my_basis.z) # Prints (0, 6, -6).
```

Comment:
`basis` is not defined here, only `my_basis` is defined. But also, why use `my_basis` here but `a` and `b` below? We should pick one convention to use for all examples.

Reply:
I was wondering the same question myself. `my_basis` is more friendly but repetitive, while using single characters would be more concise but be potentially confusing when combined with `x`, `y` and `z`.

Reply:
I'm good with either one, but after thinking about it I am leaning towards `my_basis`, since it shows the common pattern of `thing = thing.operation()`, as some users accidentally write `thing.operation()` and it's discarded.